src/Files.App/DataModels/NavigationControlItems/WslDistroItem.cs

```diff
@@ -5,30 +5,30 @@ namespace Files.App.DataModels.NavigationControlItems
 {
 	public class WslDistroItem : INavigationControlItem
 	{
-		public string Text { get; set; }
+		public string? Text { get; set; }
 
-		private string path;
+		private string? path;
 
-		public string Path
+		public string? Path
 		{
 			get => path;
 			set
 			{
 				path = value;
-				ToolTipText = Path.Contains('?', StringComparison.Ordinal) ? Text : Path;
+				ToolTipText = Path is not null ? (Path.Contains('?', StringComparison.Ordinal) ? Text : Path) : null;
```

Comment:
Writing two lines instead of one here would prevent a double ternary operator, which are a bit of a pain for the clarity of the code.